buildSrc/src/main/java/Dependencies.kt

```diff
@@ -51,9 +51,9 @@ object Versions {
     const val junit = "5.5.2"
     const val mockk = "1.12.0"
 
-    const val mockwebserver = "4.9.0"
+    const val mockwebserver = "4.10.0"
```

Comment:
Self-note: In AC, the mockwebserver and okhttp versions both need to be keep in-sync: 

https://github.com/mozilla-mobile/android-components/blob/bc36830dd90bc455cd5ef81721e0dbfab38ff16d/buildSrc/src/main/java/Dependencies.kt#L18-L20

This doesn't matter as much with fenix/focus, because we do not use okhttp directly here and mockwebserver is used for testing Glean and UI tests.

Reply:
Fun story, updating to 4.x on AC is going to need a bit of work as it's not a drop-in replacement unfortunately ☹️.